awx/api/serializers.py

```diff
@@ -5911,3 +5911,26 @@ def get_summary_fields(self, obj):
         if obj.setting:
             summary_fields['setting'] = [obj.setting]
         return summary_fields
+
+
+class AnalyticsReportSerializer(serializers.Serializer):
```

Comment:
this is just a proxy, I don't find useful to specify allowed params, which in fact belongs to Analytics API in the cloud